jobs/utils/temperature.py

```diff
@@ -0,0 +1,28 @@
+import os
+import glob
+import time
+
+class Temperature():
```

Comment:
Kolla in det här för att få ett hum om klasser:
https://www.youtube.com/watch?v=rJzjDszODTI